src/sentry/incidents/logic.py

```diff
@@ -1382,6 +1389,27 @@ def translate_aggregate_field(aggregate, reverse=False):
     return aggregate
 
 
+def trigger_sentry_app_action_creators_for_incidents(validated_alert_rule_data):
```

Comment:
This is for metric alerts, we need to fire the requesters if any SentryAppActions are present. The equivalent for issue alerts is in `rules/actions/base.py`